Sources/Valet/Internal/SecItem.swift

```diff
@@ -105,7 +104,7 @@ internal final class SecItem {
             // We should always be able to access the shared access group prefix because the accessibility of the above keychain data is set to `always`.
```

Comment:
Need to update this comment

Reply:
Why? This is still true. We'll need to update this comment as part of https://github.com/square/Valet/pull/197, which I have done. In this PR we still have `always`.